code/modules/clothing/head/misc.dm

```diff
@@ -247,3 +247,8 @@
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
 	body_parts_covered = HEAD|FACE|EARS
 	siemens_coefficient = 1
+
+/obj/item/clothing/head/beret/german
```

Comment:
Beret needs at least some basic energy armor, 25 will be fine. Also you need to write about that in description.